account_credit_control/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Account Credit Control
+======================
+
+Account Credit Control module is a part of Financial Tools used in business to
+ensure that once sales are made they are realised as cash. This module helps to
+identify outstanding debt beyond tolerance level and setup followup method.
+
+Configuration
+=============
+
+Configure the policies and policy levels in ``Accounting  > Configuration >
+Credit Control > Credit Control Policies``.
+You can define as many policy levels as you need.
+
+Configure a tolerance for the Credit control and a default policy
+applied on all partners in each company, under the Accounting tab.
+
+You are able to specify a particular policy for one partner or one invoice.
```

Comment:
I just copied the readme of your PR. Is it ok now?

Reply:
So the problem seems to be that the feature has been lost. Can you check it, @mreficent ?

Reply:
I just checked. In v11 you also can't specify a particular policy for one invoice. It's because in the `account.invoice` form you have `<field name="credit_policy_id" attrs="{'invisible': [('credit_policy_id', '=', False)]}"/>`  😂 So you can never fill this field if the field is invisible, because by default it's always False.

Reply:
@pedrobaeza @cubells any recommendation in how to amend this? I mean, I could put a default, I could delete or change the invisible domain, or maybe other things. What you propose?